aws/logs_monitoring/README.md

```diff
@@ -1,188 +1,41 @@
-**IMPORTANT NOTE: When upgrading, please ensure your forwarder Lambda function has [the latest Datadog Lambda Layer installed](https://github.com/DataDog/datadog-serverless-functions/tree/master/aws/logs_monitoring#3-add-the-datadog-lambda-layer).**
-
 # Datadog Forwarder
 
-AWS Lambda function to ship logs and metrics from ELB, S3, CloudTrail, VPC, CloudFront, and CloudWatch logs to Datadog
+AWS Lambda function to ship logs from S3 and CloudWatch, custom metrics and traces from Lambda functions to Datadog.
 
 ## Features
 
-- Forward logs through HTTPS (defaulted to port 443)
-- Use AWS Lambda to re-route triggered S3 events to Datadog
-- Use AWS Lambda to re-route triggered Kinesis data stream events to Datadog, only the Cloudwatch logs are supported
-- Cloudwatch, ELB, S3, CloudTrail, VPC and CloudFront logs can be forwarded
-- SSL Security
-- JSON events providing details about S3 documents forwarded
-- Structured meta-information can be attached to the events
-- Scrubbing / Redaction rules
-- Filtering rules (`INCLUDE_AT_MATCH` and `EXCLUDE_AT_MATCH`)
-- Multiline Log Support (S3 Only)
-- Forward custom metrics from logs
-- Submit `aws.lambda.enhanced.*` Lambda metrics parsed from the AWS REPORT log: duration, billed_duration, max_memory_used, estimated_cost
-
-## Quick Start
-
-The provided Python script must be deployed into your AWS Lambda service to collect your logs and send them to Datadog.
-
-### 1. Create a new Lambda function
-
-1. [Navigate to the Lambda console](https://console.aws.amazon.com/lambda/home) and create a new function.
-2. Select `Author from scratch` and give the function a unique name: `datadog-log-monitoring-function`
-3. For `Role`, select `Create new role from template(s)` and give the role a unique name: `datadog-log-monitoring-function-role`
-4. Under Policy templates, select `s3 object read-only permissions`.
-
-### 2. Provide the code
-
-1. Copy paste the code of the Lambda function from the `lambda_function.py` file.
-2. Set the runtime to `Python 2.7`, `Python 3.6`, or `Python 3.7`
-3. Set the handler to `lambda_function.lambda_handler`
-
-### 3. Add the Datadog Lambda Layer
-The [Datadog Lambda Layer]((https://github.com/DataDog/datadog-lambda-layer-python)) **MUST** be added to the log forwarder Lambda function. Use the Lambda layer ARN below, and replace `<AWS_REGION>` with the actual region (e.g., `us-east-1`), `<PYTHON_RUNTIME>` with the runtime of your forwarder (e.g., `Python27`), and `<VERSION>` with the latest version from the [CHANGELOG](https://github.com/DataDog/datadog-lambda-layer-python/blob/master/CHANGELOG.md).
-
-```
-arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<PYTHON_RUNTIME>:<VERSION>
-```
-
-For example:
-
-```
-arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Python37:8
-```
-
-
-### 4. Set your Parameters
-
```

Comment:
Why removing the parameters doc?

Reply:
I'm planning to copy over the doc to the parameters' description: https://github.com/DataDog/datadog-serverless-functions/pull/171/files#diff-7463e3e489db7e82f7ac591389236943R53, so it's only documented in one place (the cloudformation template and thus reflected in the stack creation/update user interface).